Tests/BackendIntegrationTests/BaseBackendIntegrationTests.swift

```diff
@@ -160,18 +160,18 @@ private extension BaseBackendIntegrationTests {
     }
 
     func verifyPurchasesDoesNotLeak() {
+        weak var purchases = Purchases.shared
+
         // See `addTeardownBlock` docs:
         // - These run *before* `tearDown`.
         // - They run in LIFO order.
-        self.addTeardownBlock { [weak purchases = Purchases.shared] in
```

Comment:
Capturing a `weak` is probably broken.

Reply:
Hmm can you explain a bit more about this?

Reply:
It’s just a guess based on the crash stack trace

Reply:
I’ll file a bug with Apple

Reply:
https://github.com/apple/swift/issues/67361